tensorly/tenalg/proximal.py

```diff
@@ -248,3 +248,207 @@ def hals_nnls(UtM, UtU, V=None, n_iter_max=500, tol=10e-8,
                 break
 
     return V, rec_error, iteration, complexity_ratio
+    
+
+def fista(AtB, pseudo_inverse, x=None, n_iter_max=100, non_negative=True, gradient_step=None,
+          sparsity_coefficient=None):
+    """
+    Fast Iterative Shrinkage Thresholding Algorithm (FISTA)
+
+    Computes and approximate solution for Ax=b linear system.
+
+    Parameters
+    ----------
+    AtB: ndarray
+       Pre-computed product of the transposed of A and B.
+    pseudo_inverse: ndarray
+       Pre-computed product of the transposed of A and A.
+    x: initialized array
+       Default: None
+    n_iter_max : int
+        Maximum number of iteration
+        Default: 100
+    non_negative : bool, default is False
+                   if True, result will be non-negative
+    gradient_step : float
```

Comment:
we could call this learning_rate or lr for consistency with opt. frameworks?

Reply:
Now it is "lr"